examples/ssh/windows_monitor_iplatency.js

```diff
@@ -0,0 +1,109 @@
+/**
+ * Domotz Custom Driver 
+ * Name: Windows Monitor IP Latency
+ * Description: This script is designe to ping an IP address and retrieves the average latency and packet loss percentage.
```

Comment:
Please also mention that the ping is executed from a windows host

There is a typo "designe"